map/place_page_info.cpp

```diff
@@ -56,7 +56,21 @@ void Info::SetFromFeatureType(FeatureType & ft)
   {
     m_uiTitle = GetBookmarkName();
 
-    auto const secondaryTitle = m_customName.empty() ? m_primaryFeatureName : m_customName;
+    std::string secondaryTitle;
+
+    if (!m_customName.empty())
+    {
+      secondaryTitle = m_customName;
+    }
+    else if (!out.secondary.empty())
+    {
+      secondaryTitle = out.secondary;
+    }
+    else
+    {
+      secondaryTitle = m_primaryFeatureName;
+    }
+
     if (m_uiTitle != secondaryTitle)
       m_uiSecondaryTitle = secondaryTitle;
```

Comment:
```suggestion
    if (!m_customName.empty())
      secondaryTitle = m_customName;
    else if (!out.secondary.empty())
      secondaryTitle = out.secondary;
    else
      secondaryTitle = m_primaryFeatureName;

    if (m_uiTitle != secondaryTitle)
      m_uiSecondaryTitle = std::move(secondaryTitle);
```

Reply:
Ok, I will fix it. ObjC style guide forbids it, but if it's ok in cpp - no problem.

Reply:
Please check our docs folder, there is a guide for C++ style there.

Reply:
Done.